deployer/health_check_tests/test_hub_health.py

```diff
@@ -46,6 +46,10 @@ async def check_hub_health(hub_url, test_notebook_path, service_api_token):
             hub_url,
             test_notebook_path,
             username=username,
+            user_options={
+                "token": service_api_token,
+                "image": "quay.io/2i2c-projectpythia/binderhub-ui-binder-2dexamples-2drequirements-55ab5c:50533eb470ee6c24e872043d30b2fee463d6943f",
```

Comment:
This should be special-cased for hubs not requiring authentication because of where the Spawner expects to find the token:

https://github.com/2i2c-org/infrastructure/blob/6214ce186ef41c79b62f057073d92d456c9ca998/helm-charts/basehub/values.yaml#L901-L910